docs/how-to/create-custom-image-using-existing-docker-image.md

```diff
@@ -0,0 +1,202 @@
+## How to create customized Docker images using an existing docker image.
+
+**WARNING**
+
+The recommended way of building extensions is to use [Rest APIs](https://api-explorer.alfresco.com/api-explorer). The AMP extension mechanism may be replaced in the future.
+
+#### Prerequisites
+
+* The extension is packaged inside an AMP
+* Docker
+* Access to Docker registry where the new Docker image will be pushed
+
+### Amps that don`t require additional configurations (easy).
+
+**Note:** We are going to use a fork of the [alfresco-bulk-import tool](https://github.com/Epurashu/alfresco-bulk-import), this is compatible with Alfresco 6.0.X(enterprise)/6.0.3(community) and above.
+### Steps:
+* We are going to use the following folder and file structure:
+
+```bash
+alfresco-custom-image                          (Dir)
+│
+└───Dockerfile                                (File)
+│
+└───amps                                       (Dir)
+    │  
+    └───alfresco-bulk-import-*.amp            (File)
+```
+* Clone and download alfresco-bulk-import project.
+
+* Trigger a Maven build from the root folder of the project (SOMEPATH/alfresco-bulk-import).
+
+```bash
+mvn clean install
+```
+
+* Copy SOMEPATH/alfresco-bulk-import/amp/target/alfresco-bulk-import-*.amp under alfresco-custom-image/amps/.
+
+* Create Dockerfile under alfresco-custom-image/
+
+* Add the following lines to the Dockerfile:
+
+```bash
+### We are going to apply amp(s) on the latest community repository image.
+FROM alfresco/alfresco-content-repository-community:latest
+
+### Copy the amps from build context to the appropriate location for your application server
+COPY amps /usr/local/tomcat/amps
+
+### Install amps on alfresco
+RUN java -jar /usr/local/tomcat/alfresco-mmt/alfresco-mmt*.jar install \
+              /usr/local/tomcat/amps/alfresco-bulk-import-*.amp /usr/local/tomcat/webapps/alfresco -nobackup -force
+```
+
+* Open a Terminal and run the following command to build the custom docker image.
+
+```bash
+docker build . -t customrepository/alfresco-custom-image:customTag
+```
+
+* The new docker image was build locally, now it can be pushed to customrepository by running the following command in Terminat/CMD:
+
+```bash
+docker push customrepository/alfresco-custom-image:customTag
+```
+
+** Note: ** It is not necessary to build/have amps locally and copy them to the new docker image, if the .amp can be downloaded via network (FTP/HTTP/HTTPS), the COPY command can be replaced by CURL/WGET command(s).
```

Comment:
Suggested change - if it make sense and doesn't change the context of what you were saying: 
** Note: ** You don’t need to build or have access to the AMPs locally before copying them to the new docker image. If you can download the `.amp` file via a network (FTP/HTTP/HTTPS), you can replace the COPY command with CURL/WGET command(s).